src/coreclr/tools/Common/Internal/Metadata/NativeFormat/MdBinaryReaderGen.cs

```diff
@@ -5,12 +5,8 @@
 
 #pragma warning disable 649
 
-using System;
```

Comment:
These files with the *Gen.cs suffix are also generated. The generator is at https://github.com/dotnet/runtime/blob/main/src/coreclr/tools/Common/Internal/Metadata/NativeFormat/Generator/NativeFormatGen.csproj

Reply:
Fixed most of the generated changes, made a comment about some primitive types generation in https://github.com/dotnet/runtime/pull/74825#discussion_r960056292